__mocks__/react-i18next.tsx

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+/* eslint-disable react/prop-types */
```

Comment:
Given that typescript covers our prop-types needs, should we just turn off this linting rule?

Reply:
Good point, will add